steps/04-install-extras.sh

```diff
@@ -27,10 +28,18 @@ case "$OS" in
       git clone https://github.com/emscripten-core/emsdk.git
     fi
     cd emsdk
-    ./emsdk install ${EMSDK_VERSION:-latest}
-    ./emsdk activate ${EMSDK_VERSION:-latest}
+    ./emsdk install ${EMSDK_VERSION:-4.0.11}
```

Comment:
I have changed this back to a static version, to make sure that `emscripten.patch` applies.